frankenphp.c

```diff
@@ -165,6 +170,27 @@ static int frankenphp_worker_request_startup() {
 	return retval;
 }
 
+PHP_FUNCTION(frankenphp_finish_request) { /* {{{ */
+    if (zend_parse_parameters_none() == FAILURE) {
+    		RETURN_THROWS();
+    }
+
+    frankenphp_server_context *ctx = SG(server_context);
+
+    if(!ctx->finished) {
+    	php_output_end_all();
+    	php_header();
+
+    	go_frankenphp_worker_handle_request_end(ctx->current_request, false);
```

Comment:
I'm not 100% sure about this function, but it appears to be what is required for both worker/non-worker mode. @dunglas, I know a bit about Go to be dangerous, but I'm not an expert. I'm going to have to rely on you if this is wrong.